llmfoundry/models/mpt/modeling_mpt.py

```diff
@@ -366,6 +366,7 @@ def _fsdp_wrap_fn(
 ) -> bool:
     # FSDP Wrap function for MPT Models
     if hasattr(module, '_fsdp_kwargs_dict'):
+        assert isinstance(module._fsdp_kwargs_dict, bool)
```

Comment:
this is not a bool, its a dict (the return type is wrong)

Reply:
Just going to `type: ignore` this unless you want me to convert this and the dependent functions to all return a dict...